applications/sasquatch/values-idfdev.yaml

```diff
@@ -139,10 +141,13 @@ rest-proxy:
   kafka:
     topics:
       - test.next-visit
+      - lsst.tap.job-run
+      - lsst.tap.job-status
```

Comment:
Should we also run lsst.tap.job-delete now?

Reply:
Sure, I can add the `lsst.tap.job-delete`  topic as well.